### PR TITLE
Add doc to help maintainers update this library

### PR DESCRIPTION
**Pull request by chrisbanes**
- Automated the version prefix when Compose snapshot versions

### CODE REVIEW

**Review by JoseAlcerreca — APPROVED** [review]
Feature request: https://github.com/marketplace/actions/twitter-action

File: docs/updating.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Updating & releasing Accompanist
+
+This doc is mostly for maintainers.
+
+## New features & bugfixes
+All new features should be uploaded as PRs against the `main` branch. 
+
+Once merged into `main`, they should be merged into the `snapshot` branch:
```
Comment (nickbutcher) [inline docs/updating.md:8]: Can this be automated with an action?

Comment (chrisbanes) [reply docs/updating.md:8]: Too risky, the Compose APIs might have changed in the snapshot.

**Review by nickbutcher — APPROVED** [review]
One idea about automating updating snapshot with any changes landing in main but content LGTM.

**Pull request merged**